lib/resources/aws/aws_ec2_instances.rb

```diff
@@ -0,0 +1,72 @@
+# author: Christoph Hartmann
+class AwsEc2Instances < Inspec.resource(1)
+  name 'aws_ec2_instances'
+  desc 'Verifies settings for an EC2 instance'
+  supports platform: 'aws'
+  example "
+    describe_ec2_instances.where { public_ip_address !=nil } do
+      its('instance_id') { should eq [] }
```

Comment:
Note, that `its('instance_id) { should eq []}` makes Inspec spit out a list of instance ID's that failed... instead of just a rather uninformative `failed` message